### ISSUE TITLE
Decky sunshine missing.

### ISSUE THREAD

**Issue by Kordha** [s0t7x/decky-sunshine#43]
Hi there,

Since last update of my steamdeck, using decky-sunshine seems impossible in gaming mode, however it works well on the desktop.

It also disappeared from the decky-loader store.

Has anyone any idea to fix this?

**Comment by Sinkero** [comment]
First time trying to install decky sunshine, disappointed to see that it's missing ! I hope it gets back in the store !

**Comment by Kordha** [comment]
Tanks for that. I also hope it comes back soon.

**Comment by Rhomagus** [comment]
> can't get my controller to work with non-steam games though from desktop mode... so I still hope it comes back to the decky store

Non-steam games are given their own randomized long digit number. These numbers can be found in the compatdata folder. You might be able to get them to work using the corresponding number as the gameid.
